control-plane-operator/controllers/hostedcontrolplane/cvo/reconcile.go

```diff
@@ -66,6 +66,10 @@ var (
 		"0000_50_operator-marketplace_10_clusteroperator.yaml",
 		"0000_50_operator-marketplace_11_service_monitor.yaml",
 		"0000_50_cluster-ingress-operator_02-deployment-ibm-cloud-managed.yaml",
+		"0000_70_cluster-network-operator_02_rbac.yaml",
```

Comment:
can we add a comment here pointing to the PRs that fix this in the payload?

Reply:
I cleaned this a bit and we are left with two manifests to omit:
 - `0000_70_cluster-network-operator_02_rbac.yaml` is not needed since we are using an admin kubeconfig available in the management cluster.
 - `0000_70_cluster-network-operator_03_deployment-ibm-cloud-managed.yaml` is not needed since we deploy CNO from the code.
Is there anything else you think we need to do here?